### ISSUE TITLE
Recommend versioned import as the "Default"

### ISSUE THREAD

**Issue by fonsp** [pikapkg/cdn#99]
On the homepage, the unversioned import is listed **first**, and it is called **Default** 
([screenshot](https://user-images.githubusercontent.com/6933510/103538031-c361eb00-4e95-11eb-8ca2-bb400c7f9a12.png)). Similarly, when viewing the landing page for a package, the unversioned import is suggested (
[preact example](https://user-images.githubusercontent.com/6933510/103538209-18056600-4e96-11eb-8e78-e85c9761fbba.png)).

_This might lead to inexperienced developers using this in their app (because it works!), but then those apps will break in the future._ 

## Suggestion
On the home page, list the versioned import as the first option; rename the unversioned import from "Default" to "Development" or something similar.

On a package page, you can dynamically write the latest import in the sample code, e.g. 
```js
<script type="module">
  import preact from 'https://cdn.skypack.dev/preact@10.5.9';
</script>
```

---

For comparison, this is how jsdelivr.com demonstrates its usage: https://www.jsdelivr.com . It shows a versioned import as the default, and it adds a clear warning to the unversioned import:

![image](https://user-images.githubusercontent.com/6933510/103538543-aa0d6e80-4e96-11eb-8a8a-d337dfeefe94.png)

**Comment by drwpow** [comment]
That’s a good suggestion. Like you pointed out, version-locking is important, as you don’t want versions to shift in production. For that reason we do encourage developers use **Pinned** versions of resources in production (which you can find when you visit https://cdn.skypack.dev/preact in a browser, e.g.).

We do want developers dipping their toes into Skypack for the first time to find it easy to use, which often means not knowing what versions packages are. But to your comment, finding the right way to communicate “dev” vs “prod” is something we’ll think about for the homepage.

**Comment by FredKSchott** [comment]
I agree that this is probably a better practice to encourage. I might push for semver major ranges like `preact@^10.5.9` instead of exact versions, which is what npm does by default on npm install.
